autograder/tests/test_submitty_autograding_shipper.py

```diff
@@ -149,3 +202,70 @@ def test_cannot_short_circuit_many_testcases(self):
         with open(os.path.join(TEST_DATA_DIR, 'complete_config_cpp_cats.json')) as infile:
             autograding_config = json.load(infile)
         self.assertFalse(shipper.can_short_circuit(autograding_config))
+
+  
+    """
+    Unit testing individual functions using mocks.
+    """
+    
+
+    # @pytest.fixture(autouse=True)
```

Comment:
Do we need to test standard out? This could possibly be removed since you are testing mainly for existence of log files.

Reply:
Standard out tests removed.